docs/usage/configuration-options.md

```diff
@@ -2446,6 +2446,17 @@ In those cases a feature request needs to be implemented.
 !!! warning "Warning for Maven users"
     For `minimumReleaseAge` to work, the Maven source must return reliable `last-modified` headers.
 
+    <!-- markdownlint-disable MD046 -->
+    If your custom maven source registry doesnt support `last-modified` header, like GAR (Google Artifact Registry's Maven implementation, then feasible option is to extend maven source registry url with https://repo1.maven.org/maven2 as the first item, then `currentVersionTimestamp` via `last-modified` will be taken from maven central for public dependencies.
```

Comment:
Are you sure this doesn't make a _functional_ difference? Are you sure that it applies so widely? e.g. does it only apply if your maven registry is a full proxy/pull-through of maven central in the first place? Otherwise you could get versions on Maven Central which aren't in your own registry

Reply:
you are right, it makes functional difference if custom maven registry is not full proxy or pull through. although in that case it doesn't make sense to combine both in the same `match` rule. perhaps its worth it to specify explicitly?

Reply:
clarified that `[`45370dc` (#35025)](https://github.com/renovatebot/renovate/pull/35025/commits/45370dc182c3330c1b672daf8b58da0353ef5db0)`

Reply:
If your proxy is only for pull-through then you could leave it out of registryUrls completely, right? You'd only need to keep it in if it's a combination of public Maven Central + private packages.

Reply:
yes, that's what we have, it will be good to clarify that. I've removed full proxy and left only pull-through